oauth2_http/java/com/google/auth/oauth2/UserCredentials.java

```diff
@@ -35,41 +35,44 @@
 import static com.google.auth.oauth2.OAuth2Utils.UTF_8;
 import static com.google.common.base.MoreObjects.firstNonNull;
 
-import com.google.api.client.http.GenericUrl;
-import com.google.api.client.http.HttpRequest;
-import com.google.api.client.http.HttpRequestFactory;
-import com.google.api.client.http.HttpResponse;
-import com.google.api.client.http.UrlEncodedContent;
+import com.google.api.client.http.*;
```

Comment:
Insted of * Use fully package qualifiers

Reply:
Done